repaygithub/cactus
# CACTUS-390 :: Select canCreateOption

Ticket: https://repayonline.atlassian.net/browse/CACTUS-390

### Testing
1. Pull down this branch and start storybook. Open the "with ComboBox" select story
2. Use the knobs to make sure that each of the ACs on the ticket are being met.
3. Open the "Basic Usage" story for SelectField
4. Repeat step 2
